grakn-migration/json/src/main/java/ai/grakn/migration/json/JsonMigrator.java

```diff
@@ -46,28 +46,30 @@
     private final Set<Reader> readers;
 
     public static void main(String[] args) {
-        MigrationCLI.init(args, JsonMigrationOptions::new).stream()
-                .filter(Optional::isPresent)
-                .map(Optional::get)
-                .forEach(JsonMigrator::runJson);
+        try {
+            MigrationCLI.init(args, JsonMigrationOptions::new).stream()
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .forEach(JsonMigrator::runJson);
+        } catch (Throwable throwable){
+            MigrationCLI.die(throwable.getMessage());
```

Comment:
Yeah, we can probably do that. I did wonder, however, if we should log the whole error to the logs and print the message to system.err. Do you think that would be helpful? (In which case, we should keep the method?)